qemu/tests/cfg/usb_device_check.cfg

```diff
@@ -9,7 +9,7 @@
     login_timeout = 180
     type = usb_device_check
     chk_usb_info_cmd = "lsusb"
-    RHEL.7, RHEL.8:
+    RHEL.7, RHEL.8, RHEL.9:
```

Comment:
Suggest use `chk_usb_info_cmd = "lsusb -v"` as the default setting, compatible with the new version.

Reply:
It's fine with me, but I want to confirm it with @ybduan , what do you think ?